tests/e2e/operator.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+#
+# Copyright 2022 Red Hat
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir="$(dirname "$(readlink -f "$0")")"
+project_dir="$(readlink -f ${script_dir}/../..)"
+
+source "${script_dir}/lib.sh"
+
+# The operator namespace.
+readonly op_ns="confidential-containers-system"
+# There should be a registry running locally on port 5000.
+export IMG=localhost:5000/cc-operator
+
+# Build the operator and push images to a local registry.
+#
+build_operator () {
+	start_local_registry
+
+	pushd "$project_dir" >/dev/null
+	make docker-build
+	make docker-push
+	popd >/dev/null
+}
+
+# Install the operator.
+#
+install_operator() {
+	start_local_registry
+
+	# The node should be 'worker' labeled
+	local label="node-role.kubernetes.io/worker"
+	if ! kubectl get node "$(hostname)" -o jsonpath='{.metadata.labels}' \
+		| grep -q "$label"; then
+		kubectl label node "$(hostname)" "$label="
+	fi
+
+	pushd "$project_dir" >/dev/null
+	make install
+	# TODO: need any checking after install?
+	make deploy
+	popd >/dev/null
+
+	# Wait the operator controller to be running.
+	local cmd="kubectl get pods -n "$op_ns" --no-headers |"
+	cmd+="egrep -q cc-operator-controller-manager.*'\<Running\>'"
+	if ! wait_for_process 120 10 "$cmd"; then
+		echo "ERROR: operator-controller-manager pod is not running"
+		return 1
+	fi
+}
+
+# Install the CC runtime.
+#
+install_ccruntime() {
+	pushd "$project_dir" >/dev/null
+	kubectl create -f config/samples/ccruntime.yaml
+	popd >/dev/null
+
+	local pod=""
+	local cmd=""
+	for pod in cc-operator-daemon-install cc-operator-pre-install-daemon; do
+		cmd="kubectl get pods -n "$op_ns" --no-headers |"
+		cmd+="egrep -q ${pod}.*'\<Running\>'"
+		if ! wait_for_process 600 30 "$cmd"; then
+			echo "ERROR: $pod pod is not running"
+			return 1
+		fi
+	done
+
+	# TODO: check the runtime is up.
+	# kubectl get runtimeclass
+}
+
+# Start a local registry where images can be stored.
+# The ansible playbooks should start it however it can get stopped when,
+# for example, the operator is unistalled.
+#
+start_local_registry() {
+	# TODO: allow callers to override the container name, port..etc
+	local registry_container="local-registry"
+
+	if ! curl -s localhost:5000; then
+		docker start local-registry >/dev/null
+		local cnt=0
+		while ! curl -s localhost:5000 -o $cnt -lt 5; do
+			sleep 1
+			cnt=$(($cnt+1))
+		done
+		[ $cnt -ne 5 ]
+	fi
+}
+
+# Uninstall the operator and ccruntime.
+#
+uninstall_operator() {
+	pushd "$project_dir" >/dev/null
+	make uninstall
```

Comment:
@wainersm, can you please try adding the following `ignore-not-found=true`  to both `uninstall` and `undeploy` target. 
eg `make uninstall ignore-not-found=true`

This will help avoid making the change recommended in `https://github.com/confidential-containers/operator/pull/70` and keep the Makefile aligned to operator-sdk defaults.

If it doesn't work, then we resort to the change mentioned in PR 70.
cc @jensfr

Reply:
as discussed in #70 I think adding `ignore-not-found=true`  is the correct change to make. Thanks @bpradipt !

Reply:
@bpradipt @jensfr implement a test case for the uninstall operation in https://github.com/confidential-containers/operator/pull/63/commits/b535a90e55745c1afa194dd0ea4c83d91ebabf86 . I added the `ignore-not-found=true` to the Make calls as you suggested.